docs/user/dql/functions.rst

```diff
@@ -2330,6 +2330,31 @@ Example::
     +---------------------+---------------------+
 
 
+POSITION
+------
+
+Description
+>>>>>>>>>>>
+
+Usage: The syntax POSITION(substr IN str) returns the position of the first occurrence of substring substr in string str. Returns 0 if substr is not in str. Returns NULL if any argument is NULL.
```

Comment:
Returns 0 if substr is not in str? not -1?
what if POSITION('world' IN 'world')?

Reply:
This function is the synonym of the `LOCATE` function. And works the same way in mySQL and legacy engine.
It returns 0, if substr in not in string like `LOCATE` and `POSITION` do in mySql and legacy engine.
`POSITION('world' IN 'world')` would return 1 as the first argument matches the given string (2nd arg) starting from the 1st position.
<img width="627" alt="Screenshot 2022-12-06 at 8 56 55 AM" src="https://user-images.githubusercontent.com/116052492/205974483-bf102e6d-c231-4fb6-927a-5deb755b2997.png">